packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/index.tsx

```diff
@@ -32,32 +31,30 @@ const ConfirmResetPassword = (_: RouteComponentProps) => {
   logPageViewEvent(viewName, REACT_ENTRYPOINT);
 
   const navigate = useNavigate();
-  let { state = {} } = useLocation();
-  const account = useAccount();
-  const [passwordResetResend, setPasswordResetResend] = useState(false);
-  const [isPolling, setIsPolling] = useState<number | null>(
-    POLLING_INTERVAL_MS
-  );
-  
-  const navigateToPasswordReset = useCallback(() => {
-    navigate('reset_password?showReactApp=true', { replace: true });
-  }, [navigate]);
+  let { state } = useLocation();
 
   if (!state) {
     state = {};
   }
 
-  let { email, passwordForgotToken } = state as ConfirmResetPasswordProps;
+  const { email, passwordForgotToken } =
+    state as ConfirmResetPasswordLocationState;
 
-  if (!email || !passwordForgotToken) {
-    navigateToPasswordReset();
-  }
+  const account = useAccount();
+  const [passwordResetResend, setPasswordResetResend] = useState(false);
```

Comment:
very optional nit, but what would you think about renaming this to `passwordResetResendIsVisible, setPasswordResetResendIsVisible` ?

Reply:
Good catch, this will be changed in the PR for https://mozilla-hub.atlassian.net/browse/FXA-6891 👍